src/testapp/scenarios.cpp

```diff
@@ -319,18 +319,107 @@ static ScenarioMap_t make_scenarios()
         commonScene     = setup_common_scene        (builder, rTopData, scene, application, defaultPkg);
         physics         = setup_physics             (builder, rTopData, scene, commonScene);
         physShapes      = setup_phys_shapes         (builder, rTopData, scene, commonScene, physics, sc_matPhong);
-        terrain         = setup_terrain             (builder, rTopData, scene);
+        terrain         = setup_terrain             (builder, rTopData, scene, commonScene);
         terrainIco      = setup_terrain_icosahedron (builder, rTopData, terrain);
         terrainSubdiv   = setup_terrain_subdiv_dist (builder, rTopData, scene, terrain, terrainIco);
 
+        OSP_DECLARE_GET_DATA_IDS(terrain,    TESTAPP_DATA_TERRAIN);
+        auto &rTerrain = top_get<ACtxTerrain>(rTopData, idTerrain);
+        auto &rTerrainFrame = top_get<ACtxTerrainFrame>(rTopData, idTerrainFrame);
+
         initialize_ico_terrain(rTopData, terrain, terrainIco, {
-            .radius                 = 50.0,
-            .height                 = 5.0,
+            .radius                 = 6371000.0, // Earth radius
+            .height                 = 20000.0,   // Height between Mariana Trench and Mount Everest
+            .skelPrecision          = 10,        // 2^10 units = 1024 units = 1 meter
+            .skelMaxSubdivLevels    = 19,
+            .chunkSubdivLevels      = 4
+        });
+
+        // Set scene position relative to planet to be just on the surface
+        rTerrainFrame.position = Vector3l{0,0,6371000} * 1024;
```

Comment:
minor: can the earth radius be extracted out to a constexpr variable for self-documentation purposes?